### PR TITLE
 OpenPaaS-Suite/esn-frontend-mailto#55 remove mailtoParser service

### PR DESCRIPTION
**Pull request by MichaelBailly**
Merge https://github.com/OpenPaaS-Suite/esn-frontend-mailto/pull/56 first.

Closes OpenPaaS-Suite/esn-frontend-mailto#55

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "OpenPaaS suite Webmail (https://open-paas.org/)",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/karma/bin/karma start test/config/karma.conf.js",
+    "test": "LANG=C karma start test/config/karma.conf.js",
```
Comment (tuanlc) [inline package.json:7]: Can you explain why? And this deserves a dedicated commit, IMO ;)

Comment (MichaelBailly) [reply package.json:7]: ah, Crap, your eyes are everywhere !
My Linux system is using FR language, so the launched firefox also, and it makes some tests break, because we check the result (HTML), and compare it using EN language.

Comment (MichaelBailly) [reply package.json:7]: commits split

**Review by rezk2ll — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Review by tuanlc — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]
